### PR TITLE
Update link to RequireJS migration article

### PR DESCRIPTION
**Pull request by sgrossberndt**
Original source is no longer available, link to archive.org instead

### CODE REVIEW

File: Documentation/ApiOverview/Backend/JavaScript/ES6/Index.rst [hunk 1]
```diff
@@ -123,7 +123,7 @@ Migration from RequireJS
 
 ..  note:: For general information on how to migrate from AMD (RequireJS) to
     ES6 see this article: `Arthur Yidi: Migrate JavaScript Modules From AMD
-    to ES6 <https://arthuryidi.com/migrate-amd-modules/>`__. This is about
+    to ES6 <https://web.archive.org/web/20220516095256/https://arthuryidi.com/migrate-amd-modules/>`__. This is about
```
Comment (linawolf) [inline Documentation/ApiOverview/Backend/JavaScript/ES6/Index.rst:126]: Something is odd about the url now?

**Comment by linawolf** [comment]
@sgrossberndt did you check if this URL needs to be adjusted in main as well? We usually try to make changes on main and then backport them to 12.4 automatically. It is not possible to automatically preport to main

File: Documentation/ApiOverview/Backend/JavaScript/ES6/Index.rst (hunk 1 above)
Comment (sgrossberndt) [reply Documentation/ApiOverview/Backend/JavaScript/ES6/Index.rst:126]: The URL is valid like this. web.archive.org is a service mirroring sites. The URL points to the original site at a certain point in time when archive.org did a scan of that site.

Visit it: https://web.archive.org/web/20220516095256/https://arthuryidi.com/migrate-amd-modules/

**Comment by sgrossberndt** [comment]
> @sgrossberndt did you check if this URL needs to be adjusted in main as well? We usually try to make changes on main and then backport them to 12.4 automatically. It is not possible to automatically preport to main

Yes, I know and checked this first, the migration part is only in [12.4](https://docs.typo3.org/m/typo3/reference-coreapi/12.4/en-us/ApiOverview/Backend/JavaScript/ES6/Index.html#requirejs-migration) but not in [main](https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/Backend/JavaScript/ES6/Index.html)

**Review by linawolf — APPROVED** [review]

**Pull request merged**